### ISSUE TITLE
Incompatible contract descriptor

### ISSUE THREAD

**Issue by claudiosdc** [mycitadel/mycitadel-swiftui#2]
I have noticed that the My Citadel app (installed on my iPhone) is showing a descriptor text in the "Share descriptor" section of the "Contract details" panel (right below the QR code) that is not compatible with the Bitcoin output descriptor reference found [here](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md#reference).

This is the descriptor text displayed on the My Citadel app:

```shell
wpkh(m=[f7e41fce]/827166'/0'/1'/1537404484'/802918955'=[tpubDGDcjfCNU5T8FBXVquV8TqM8WZyLh3WGEq4CQB3heHzwDeXUGwKuF1QMiueug5vYimpNhnGGtbkHDWvXXL4mmshNYgXUsi2KXZvNu5o5JxD]/0/*)#tgh8pjqe
```

And this was the expected descriptor text compatible with the reference doc:

```shell
wpkh([f7e41fce/827166'/0'/1'/1537404484'/802918955']tpubDGDcjfCNU5T8FBXVquV8TqM8WZyLh3WGEq4CQB3heHzwDeXUGwKuF1QMiueug5vYimpNhnGGtbkHDWvXXL4mmshNYgXUsi2KXZvNu5o5JxD/0/*)#xxxxxxxx
```

Where `#xxxxxxxx` is the appropriately calculated checksum.

**Comment by dr-orlovsky** [comment]
Yes, this is an intentional extension of the standard. You can manually remove `m=` prefix and checksum and it will work with Bitcoin Core. But you should not do that since bitcoin Core is RGB unaware and will not see all utxos (some of them are tweaked)
